extensions/amp-story/1.0/amp-story-draggable-drawer-header.css

```
@@ -90,6 +94,12 @@
   top: 0 !important;
 }
 
+[theme="dark"].i-amphtml-amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header {
+  --i-amphtml-draggable-drawer-background-color: #202125 !important;
+  --i-amphtml-draggable-drawer-text-color: white !important;
```

To stay consistent with hex values (here and above)

```suggestion
  --i-amphtml-draggable-drawer-text-color: #ffff !important;
```